### PR TITLE
Move Phone model to the correct app while keeping data

### PR DESCRIPTION
**Pull request by mick88**
## Description
Fixes issue #519 by adding state migration to the phone plugin app. This MR moves the model to the same app where model code resides.
Note that all operations in this MR are purely state migrations and no database operations are carried out therefore there is no risk of data loss. The table name is hardcoded as to avoid renaming database table following the model being moved to a different app.

## Motivation and Context
As documented in issue #519, django does not correctly create the phone model as the model code is in a different app from where the model is assigned to. 

## How Has This Been Tested?
Migrations were ran. Running makemigrations shows that there are no outstanding model changes.

## Types of changes

- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: two_factor/migrations/0008_delete_phonedevice.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.4 on 2022-08-20 19:03
```
Comment (claudep) [inline two_factor/migrations/0008_delete_phonedevice.py:1]: I think you could safely remove those Generated lines, in both files.

Comment (mick88) [reply two_factor/migrations/0008_delete_phonedevice.py]: done

**Comment by claudep** [comment]
Could you still reformat the new initial migration so flake8 is happy (line length <= 119)?

**Review by claudep — APPROVED** [review]
Thanks!

**Pull request merged**